### PR TITLE
Add disk metrics to list of docker container metrics

### LINKED ISSUE #4203: [Request]: Document network and disk metrics in infrastructure UI
### Description

https://github.com/elastic/observability-docs/pull/4186 added documentation about monintoring container metrics for version 8.15. 

8.16 adds new metrics that need to be documented for containers, specifically:

* Network
* Disk

Doc changes required:

- [x] The screen capture metrics-overlay-containers.png needs to be updated to show tabs for the new metrics.
- [x] The intro for the section about the Overview tab in container-details-partial.asciidoc should mention network and disk metrics.
- [x] container-metrics.asciidoc needs to be updated with descriptions of network and disk metrics (see [this document](https://docs.google.com/document/d/1xrLMnT-Nz5TqkPT5PxNjJNLjrU4qXMsbyTWkmqLhtLw/edit#heading=h.lrmgncaurmh9))

Based on [this comment](https://github.com/elastic/observability-docs/pull/4186#issuecomment-2311864167), it sounds like we might need to wait for dev input before documenting Kubernetes network and disk metrics.

### Resources

Related PR: https://github.com/elastic/observability-docs/pull/4186

### Which documentation set does this change impact?

Stateful and Serverless

### Feature differences

feature is the same, but will be delivered at different times.

### What release is this request related to?

8.16

### Collaboration model

The documentation team

### Point of contact.

**Main contact:** @roshan-elastic @jennypavlova 

**Stakeholders:** n/a

### PR DESCRIPTION
**Pull request by dedemorton**
## Description
Adds disk metrics and calculations to the reference topic about container metrics. (Note that the network metrics were already documented but did not include the calculations.)

Preview links:

Stateful: https://observability-docs_bk_4396.docs-preview.app.elstc.co/guide/en/observability/master/container-metrics.html

Serverless: https://observability-docs_bk_4396.docs-preview.app.elstc.co/guide/en/observability/master/container-metrics.html

### Documentation sets edited in this PR

_Check all that apply._

- [x] Stateful (`docs/en/observability/*`)
- [x] Serverless (`docs/en/serverless/*`)
- [ ] Integrations Developer Guide (`docs/en/integrations/*`)
- [ ] None of the above

### Related issue
Closes #4203 

## Checklist

- [ ] Product/Engineering Review
- [x] Writer Review

### Follow-up tasks

_Select one._

* This PR does _not_ need to be ported to another doc set because:
  - [ ] The concepts in this PR only apply to one doc set (serverless _or_ stateful)
  - [x] The PR contains edits to both doc sets (serverless _and_ stateful)
* This PR needs to be ported to another doc set:
  - [ ] Port to stateful docs: \<link to PR or tracking issue>
  - [ ] Port to serverless docs: \<link to PR or tracking issue>

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-hzu1x0vy5-elastic-dev.vercel.app

* Built with commit 84d399ee7609486912c6bb365bbf77fd062e3499

> Issues? Visit #next-docs in Slack

File: docs/en/serverless/transclusion/container-details.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <DocAccordion buttonContent="Overview">
 
-The **Overview** tab displays key metrics about the selected container, such as CPU usage and memory usage.
+The **Overview** tab displays displays key metrics about the selected container, such as CPU, memory, network, and disk usage.
```
Comment (dedemorton) [inline docs/en/serverless/transclusion/container-details.mdx:5]:
```suggestion
The **Overview** tab displays key metrics about the selected container, such as CPU, memory, network, and disk usage.
```

**Review by bmorelli25 — APPROVED** [review]
Learning a lot about metrics today

**Review by roshan-elastic — APPROVED** [review]
LGTM!

**Pull request merged**